platforms/openstack/modules/vms/vms.tf

```diff
@@ -108,6 +109,15 @@ resource "openstack_compute_instance_v2" "nostorage" {
 
   metadata = "${merge(module.tags.value, map("Name", "${var.prefix}-${var.node_type}-${count.index}"))}"
 
+  block_device {
```

Comment:
I would comment it out, since OpenStack seems to ignore the size attribute on the root disk.